src/content/en/fundamentals/design-and-ui/_toc.yaml

```diff
@@ -15,6 +15,8 @@ toc:
         path: /web/fundamentals/design-and-ui/media/video
       - title: Multi-Device Content
         path: /web/fundamentals/design-and-ui/responsive/content
+    - title: Basics of UX
```

Comment:
Can you move this up so that it's under the Overview, I think it's important enough to warrant near top billing.